docs/modules/debug/require-inspector.md

```diff
@@ -0,0 +1,35 @@
+<%( TOC )%>
+
+# Require Inspector
```

Comment:
is this the name we settled on? 

`Require Inspector` vs `Requirable Inspector`